pkg/workloads/cortex/lib/storage/local.py

```diff
@@ -141,3 +141,13 @@ def download_and_unzip(self, key, local_dir):
         local_zip = os.path.join(local_dir, "zip.zip")
         self.download_file(key, local_zip)
         util.extract_zip(local_zip, delete_zip_file=True)
+
+    def list_objects(self, path):
+        objects = []
+        for root, dirs, files in os.walk(path):
+            for name in files:
+                objects.append(os.path.join(root, name))
+            for name in dirs:
```

Comment:
I don't think the S3 version of `list_objects()` returns directories, just objects. Can you confirm, and if so, only return files here?